### PR TITLE
Adds support for coop login token, fixes #583

### LINKED ISSUE #583: Issues with Coop login
Coop Login does not work.

### PR DESCRIPTION
**Pull request by fredrike**

### CODE REVIEW

**Comment by goober** [comment]
Do you mind adding a comment in the CHANGELOG as well?

**Comment by fredrike** [comment]
Added a line in CHANGELOG too..

**Comment by fredrike** [comment]
Added comment in CHANGELOG.

File: CHANGELOG [hunk 1]
```diff
@@ -1,8 +1,11 @@
 Please view this file on the master branch, on stable branches it's out of date.
 
+pending
+* Fixes for Coop login.
+
 v1.9.10.1 (2015-11-22)
 * Update certificate for ICA.
-* Fixes bug where banks without password could not be added.
+pt:        pass bug where banks without password could not be added.
```
Comment (goober) [inline CHANGELOG]: These lines should not be included in the PR
